server/tests/batch_comparison/test_batch_inventory.py

```diff
@@ -316,7 +316,7 @@ def test_batch_inventory_happy_path(
         json.loads(rv.data),
         {
             "file": {
-                "name": asserts_startswith("batchTallies"),
+                "name": asserts_startswith("batch_tallies"),
```

Comment:
Nit: should we be consistent with file prefixes using either hyphen or underscore word separators? I noticed there's a mix